tripy/tripy/frontend/trace/ops/slice.py

```diff
@@ -256,8 +256,33 @@ def clamp_bound(bound: Union[int, Tensor]) -> Union[int, Tensor]:
     return out
 
 
-# Because the helper is called inside another function, we need to skip one entry in the call stack to find
-# the original call to user code.
-@frontend_utils.convert_to_tensors(skip_num_stack_entries=1)
+@frontend_utils.convert_to_tensors()
 def slice_helper(tensor, *slice_params: TensorLike):
+    from tripy.utils import get_arg_candidate_column_offsets
+
+    # The default behavior of convert_to_tensors will not add the correct column info to the slice params
+    # because this call occurs *inside* the overridden call to __getitem__, so we adjust the column info manually.
+    arg_names = ["tensor"] + ["slice_params"] * len(slice_params)
+    for arg_index, arg in enumerate(slice_params):
+        # Look for the call to __getitem__. We need to go one stack frame beyond to get to the *user* call of __getitem__.
+        frame_index = -1
+        for idx, source_info in enumerate(arg.stack_info):
+            if source_info._dispatch_target == "__getitem__":
+                frame_index = idx + 1
+                break
```

Comment:
I think we can lift this out of the loop right? The frame for `__getitem__` should be the same for all the arguments to slice.

Reply:
Oh I had overlooked that. That's correct.